core/src/main/resources/hudson/views/Messages_pl.properties

```diff
@@ -19,5 +19,12 @@
 # LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 # THE SOFTWARE.
-
 JobColumn.DisplayName=Nazwa
+LastSuccessColumn.DisplayName=Ostatni sukces
+LastDurationColumn.DisplayName=Ostatni czas trwania
+WeatherColumn.DisplayName=Prognoza
+StatusColumn.DisplayName=Status
+LastStableColumn.DisplayName=Ostatnie stabilne
+BuildButtonColumn.DisplayName=Przycisk budowania
+LastFailureColumn.DisplayName=Ostatnie niepowodzenie
+DefaultViewsTabsBar.DisplayName=Domy\u015Blny widok TabBar
```

Comment:
Given that it's less of a proper noun than project types this can probably easily be translated. The Google problem applies to most localized labels you want to learn about. Maybe the English name could be mentioned in the associated `help-*_pl.html` if there is one?

FWIW I wondered how Firefox did it, and it seems they call their similar UI element *pasku kart*.